js/generateCard.js

```diff
@@ -0,0 +1,37 @@
+import {similarAdvertisements, syncTypes} from './data.js';
+import {checkValueOfPhoto, checkValueOfElement} from './util.js';
+
+const addCardTemplate = document.querySelector('#card').content;
+const CardTemplate = addCardTemplate.querySelector('.popup');
+
+const generateAdvertisements = similarAdvertisements;
+
+generateAdvertisements.forEach((el) => {
```

Comment:
зачем тратить память на сохранение массива similarAdvertisements в generateAdvertisements если можно просто итерироваться сразу по similarAdvertisements ?
```suggestion
similarAdvertisements.forEach((el) => {
```